### ISSUE TITLE
Off topic - Is anyone seeing issues with Plex moving a recording?

### ISSUE THREAD

**Issue by CTJohnK** [tgorgdotcom/locast2plex#167]
Off topic, but thought I'd ask the group at large -

Every once in a while, Plex is failing a recording with a message "Could not move recorded media to destination".  However earlier and later records save just fine.  The recordings are all going into the same directory, there are no other recordings occurring at that time and it does not happen with the same channel.

Anyone else noticing this?

**Comment by Guyverix** [comment]
Yep!  had this happen when I was debugging getting locast2plex working.. Ended up chmod and chowning directory structures to the plex user, and that fixed the problem for me

**Comment by mr-david** [comment]
> Yep! had this happen when I was debugging getting locast2plex working.. Ended up chmod and chowning directory structures to the plex user, and that fixed the problem for me

Thank you, this pointed me in the right direction. I had two recordings that consistently failed with the "Could not move recorded media to destination" error. Turns out the directories for the two shows were not owned by the plex user and did not have write permission.
